sys/usb/usbus/usbus_fmt.c

```diff
@@ -205,6 +205,9 @@ static size_t _fmt_descriptors_iface_alts(usbus_t *usbus,
         len += sizeof(usb_descriptor_interface_t);
         usb_iface.alternate_setting = alts++;
         usb_iface.num_endpoints = _num_endpoints_alt(alt);
+        if (alt->descr) {
+            usb_iface.idx = alt->descr->idx;
+        }
```

Comment:
I think this needs an `else` that sets the `usb_iface.idx = 0;`. Same how it is done below in `_fmt_descriptors_ifaces()`. The `usb_descriptor_interface_t` is allocated on the stack and not zero'd so all members should be explicitly set.

Feel free to amend it in

Reply:
Done.
Squashed directly.